bfm/README.md

```diff
@@ -0,0 +1,25 @@
+# Business File Manager (BFM)
+
+**Anmeldedaten für gemeinsames BFM-Konto**
```

Comment:
Please translate the whole file to English. As there are English speaking participants.